articles/20240822_v0.24.0_news.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Daytona v0.24.0"
+description: "Explore the new features, bug fixes, and improvements in Daytona v0.25.0"
+date: 2024-08-22
+author: "Dev Gajjar"
+tags: ["Release Notes", "v0.24.0"]
+---
+
+# Daytona v0.24.0 Release Summary
```

Comment:
Please tell me if I need to update the task if it was confusing, the idea is to cover the office hours as an article and not version, we have https://www.daytona.io/changelog for this